### PR TITLE
Docs - Cms Kit Blogs

### PR DESCRIPTION
**Pull request by enisn**

### CODE REVIEW

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# Blog System
+
+CMS kit provides a **Blog** system to to add Blogs & Posts to public website. This section describes the details of the Blog system.
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
CMS kit provides a **blog** system to add blogs & posts to a public website. This section describes the details of the blog system.
```

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+## Feature
+
+CMS kit uses the [global feature](https://docs.abp.io/en/abp/latest/Global-Features) system for all implemented features. Commercial startup templates come with all the CMS Kit related features are enabled by default, if you create the solution with the public website option. If you're installing the CMS Kit manually or want to enable the rating feature individually, open the `GlobalFeatureConfigurator` class in the `Domain.Shared` project and place the following code to the `Configure `method.
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
CMS kit uses the [global feature](https://docs.abp.io/en/abp/latest/Global-Features) system for all implemented features. Commercial startup templates come with all the CMS Kit-related features are enabled by default, if you create the solution with the public website option. If you're installing the CMS Kit manually or want to enable the blog feature individually, open the `GlobalFeatureConfigurator` class in the `Domain.Shared` project and place the following code to the `Configure `method.
```

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+# Internals
+
+## Domain Layer
+
+#### Aggregates
+
+This module follows the [Entity Best Practices & Conventions](https://docs.abp.io/en/abp/latest/Best-Practices/Entities) guide.
+
+- `Blog` _(aggregate root)_: Presents blogs of application.
+- `BlogPost`_(aggregate root)_: Presents Blog Posts by Blogs
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
- `BlogPost`_(aggregate root)_: Presents blog posts in blogs.
```

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+- `BlogFeauture`:_(aggregate root)_: Presents blog features enabled/disabled state. Such as reactions, ratings, comments etc.
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
- `BlogFeature`:_(aggregate root)_: Presents blog features enabled/disabled state. Such as reactions, ratings, comments, etc.
```

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+- `IBlogRepository`
+- `IBlogPostRepository`
+- `IBlogFeatureRepository`
+
+#### Domain services
+
+This module follows the [Domain Services Best Practices & Conventions](https://docs.abp.io/en/abp/latest/Best-Practices/Domain-Services) guide.
+
+- `BlogManager`: Includes some operations for `Blog` aggreate root to keep data consistency.
+  - `CreateAsync`: Creates a new Blog entity and makes sure slug is used one time.
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
  - `CreateAsync`: Creates a new `Blog` entity and makes sure slug is used one time.
```

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+  - `UpdateAsync`: Updates existing Blog entity and makes sure slug is used one time.
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
  - `UpdateAsync`: Updates existing `Blog` entity and makes sure slug is used one time.
```

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+  - `SetAsync`: Sets a feature enabled/disabled for a Blog.
+  - `SetDefaultsAsync`: Resets all feature configuration for a Blog.
+
+### Application layer
+
+#### Application Services
+
+##### Admin
+
+- `BlogAdminAppService` _(implements `IBlogAdminAppService`)_: Implements use cases of Blog Management.
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
- `BlogAdminAppService` _(implements `IBlogAdminAppService`)_: Implements use cases of blog management.
```

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+- `BlogFeatureAdminAppService` _(implements `IBlogFeatureAdminAppService`)_: Implements use cases of Blog Feature Management.
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
- `BlogFeatureAdminAppService` _(implements `IBlogFeatureAdminAppService`)_: Implements use cases of blog feature management.
```

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+- `BlogPostAdminAppService` _(implements `IBlogPostAdminAppService`)_: Implements use cases of Blog Post Management.
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
- `BlogPostAdminAppService` _(implements `IBlogPostAdminAppService`)_: Implements use cases of blog post management.
```

File: docs/en/Modules/cms-kit/Blog-System.md [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+
+##### Common
+
+- `BlogFeatureAppService` _(implements I`BlogFeatureAppService`)_: Implements feature checking operations.
+
+##### Public
+
+- `BlogPostPublicAppService` _(implements `IBlogPostPublicAppService`)_: Implements public blog post displaying operations.
```
Comment (ilkayilknur) [inline docs/en/Modules/cms-kit/Blog-System.md]:
```suggestion
- `BlogPostPublicAppService` _(implements `IBlogPostPublicAppService`)_: Implements use cases of blog posts management on public websites.
```

**Review by ilkayilknur — CHANGES_REQUESTED** [review]

**Comment by ilkayilknur** [comment]
Instead of mentioning the Admin UI at the end of the document, we need to mention UI controls.  I already covered the admin UI in the overall document. We don't need to duplicate that content. @enisn

**Comment by enisn** [comment]
> Instead of mentioning the Admin UI at the end of the document, we need to mention UI controls. I already covered the admin UI in the overall document. We don't need to duplicate that content. @enisn

Blogs don't have any public UI component to be invoked. 

It has only **DefaultBlogPostCommentViewComponent** but it encapsulates **CommentingViewComponent**.

**Review by ilkayilknur — APPROVED** [review]

**Pull request merged**